client/Dockerfile

```
@@ -1,10 +1,15 @@
 # ============================================
 # Development Stage
 # ============================================
-FROM oven/bun:1.2.22-alpine AS dev
+FROM node:22-alpine AS dev
 
 WORKDIR /app
 
+# Install bun for package management (uses bun.lock)
+RUN apk add --no-cache bash curl && \
```

Hadolint (2.14.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)